docs/api/cpp/quickstart.rst

```diff
@@ -0,0 +1,188 @@
+Quickstart Guide
+================
+
+First, create a cvc5 :cpp:class:`Solver <cvc5::api::Solver>` instance:
```

Comment:
Probably not for this PR, but I wonder what do you think about adding details on how to build this example, that is, starting one step before.

Reply:
Hm. This is the quickstart guide for the API, not sure if we should have build instructions there. I'm thinking this should probably rather be mentioned in https://cvc5.github.io/docs/examples/examples.html. @nafur what do you think?